ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -206,6 +206,15 @@ VAV_SD_DSP_DFSS:
   - DSP
   - DFSS
 
+VAV_SD_CSP_DFSS:
+  description: "Fan powered VAV."
```

Comment:
please expand description: "Fan powered VAV for cooling only applications (e.g. IDF or computer rooms)."